static/app/views/dashboardsV2/types.tsx

```diff
@@ -11,6 +11,7 @@ export enum DisplayType {
 }
 
 export type WidgetQuery = {
+  id?: string;
```

Comment:
Is this change still required?

Reply:
Nope! No longer required after we switched to index instead of id for key. Updated!